### DISCUSSION TITLE
Order files by names when searching in the file picker.

### DISCUSSION THREAD

**Discussion by YohDeadfall** [helix-editor/helix#10988] [General]
When searching in the file picker the results should be ordered alphabetically if there's no difference between them. The issue can be easily reproduced in the `bevyengine/bevy-wewbsite` repository by looking for `sass`.

Actual:

```
sass/site.scss
sass/_vars.scss
sass/_print.scss
sass/_utils.scss
sass/_mixins.scss
sass/_functions.scss
sass/pages/404.scss
sass/pages/_docs.scss
sass/pages/_news.scss
sass/pages/_assets.scss
sass/pages/_donate.scss
sass/pages/_people.scss
sass/elements/_body.scss
sass/elements/_html.scss
sass/pages/_content.scss
```

Expected:

```
sass/_functions.scss
sass/_mixins.scss
sass/_print.scss
sass/_utils.scss
sass/_vars.scss
sass/elements/_body.scss
sass/elements/_html.scss
sass/pages/404.scss
sass/pages/_assets.scss
sass/pages/_content.scss
sass/pages/_docs.scss
sass/pages/_donate.scss
sass/pages/_news.scss
sass/pages/_people.scss
sass/site.scss
```

`LC_ALL` isn't set in my case.

**Comment by pascalkuthe** [comment]
files are ordered by `score -> length -> alphabetical order`. This is intentional and matches fzf.

**Reply by YohDeadfall** [reply]
Makes sense, but for huge code bases it might be an issue. It's not the first time I face that, but had more complicated scenarios. Is it possible to limit how the input is treated? Let's say, it should be a single world and not "s*ass" or "s*a*ss" which bith match "static/assets".
